mappings/net/minecraft/resource/ResourceReloader.mapping

```diff
@@ -1,30 +1,65 @@
-CLASS net/minecraft/class_4014 net/minecraft/resource/ResourceReloader
-	FIELD field_17927 manager Lnet/minecraft/class_3300;
-	FIELD field_18042 prepareStageFuture Ljava/util/concurrent/CompletableFuture;
-	FIELD field_18043 applyStageFuture Ljava/util/concurrent/CompletableFuture;
-	FIELD field_18044 waitingListeners Ljava/util/Set;
-	FIELD field_18045 listenerCount I
-	FIELD field_18046 applyingCount I
-	FIELD field_18047 appliedCount I
-	FIELD field_18048 preparingCount Ljava/util/concurrent/atomic/AtomicInteger;
-	FIELD field_18049 preparedCount Ljava/util/concurrent/atomic/AtomicInteger;
-	METHOD <init> (Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;Lnet/minecraft/class_3300;Ljava/util/List;Lnet/minecraft/class_4014$class_4047;Ljava/util/concurrent/CompletableFuture;)V
-		ARG 1 prepareExecutor
-		ARG 2 applyExecutor
-		ARG 3 manager
-		ARG 4 listeners
-		ARG 5 creator
-		ARG 6 initialStage
-	METHOD method_18369 create (Lnet/minecraft/class_3300;Ljava/util/List;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;Ljava/util/concurrent/CompletableFuture;)Lnet/minecraft/class_4014;
-		ARG 0 manager
-		ARG 1 listeners
-		ARG 2 prepareExecutor
-		ARG 3 applyExecutor
-		ARG 4 initialStage
-	CLASS class_4047 Factory
-		METHOD create (Lnet/minecraft/class_3302$class_4045;Lnet/minecraft/class_3300;Lnet/minecraft/class_3302;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
-			ARG 1 helper
-			ARG 2 manager
-			ARG 3 listener
-			ARG 4 prepareExecutor
-			ARG 5 applyExecutor
+CLASS net/minecraft/class_3302 net/minecraft/resource/ResourceReloader
+	COMMENT A resource reloader performs actual reloading in its {@linkplain #reload
+	COMMENT reload} in a reloadable resource manager it is registered to.
+	COMMENT
+	COMMENT @see ReloadableResourceManager
+	COMMENT @see SinglePreparationResourceReloader SinglePreparationResourceReloader
+	COMMENT (completes preparation in one method)
+	COMMENT @see SynchronousResourceReloader SynchronousResourceReloader
+	COMMENT (performs all reloading in the apply executor)
```

Comment:
What happened here?

Reply:
Line too long, wrapping. The rendered effect should be
`SynchronousResourceReloader (performs all reloading in the apply executor)`